Source/OxyPlot.Avalonia/Utilities/ConverterExtensions.cs

```diff
@@ -422,10 +442,14 @@ public static OxyMouseWheelEventArgs ToMouseWheelEventArgs(this PointerWheelEven
         /// <returns>A <see cref="OxyMouseEventArgs" /> containing the converted event arguments.</returns>
         public static OxyMouseDownEventArgs ToMouseDownEventArgs(this PointerPressedEventArgs e, IInputElement relativeTo)
         {
+            var point = e.GetCurrentPoint(relativeTo);
+            
             return new OxyMouseDownEventArgs
             {
-                ChangedButton = e.MouseButton.Convert(),
+                ChangedButton = point.Properties.PointerUpdateKind.Convert(),
+#pragma warning disable CS0618 // Type or member is obsolete
                 ClickCount = e.ClickCount,
```

Comment:
ClickCount is obsolete, but there is no 1to1 replacement. Although it can be replaced with DoubleTapped event.